openedx/features/specializations/helpers.py

```diff
@@ -0,0 +1,5 @@
+from datetime import datetime
+
+
+def date_from_str(date_str, date_format='%Y-%m-%dT%H:%M:%SZ'):
```

Comment:
I searched in edx-platform there is a constant defined for this format `%Y-%m-%dT%H:%M:%SZ` but it's in ecommerce app. Let's please use that if it makes sense otherwise let's please define a constant here.